src/components/GradesView/GradesView.scss

```diff
@@ -76,49 +37,21 @@
   }
 }
 
-.filter-group {
-  .grade-filter-inputs {
-    .percent-group {
-      display: inline-block;
-      .form-group, .pgn__form-group {
-        width: 115px;
-        display: inline-block;
-      }
-      .input-percent-label {
-          margin-top: 22px;
-          margin-left: 5px;
-          margin-right: 5px;
-      }
-    }
-  }
-  .grade-filter-action {
-    text-align: right;
-  }
-}
-
-.mb-85 {
-    margin-bottom: 85px;
-}
-
 .modal-dialog {
     max-width: 1000px;
 }
 
-.wrap-text-in-cell {
-    overflow-wrap: break-word;
-    word-wrap: break-word;
-}
-
 select#ScoreView.form-control {
   padding-right: 26px;
 }
 
 [dir=rtl] #course-grade-tooltip .arrow {
     right: initial;
     left: 0;
-    
+
     &:before {
         border-width: 0.4rem 0.4rem 0.4rem 0;
-        border-right-color: $black;
+        border-right-color: black;
+        border-right-color: var(--black);
```

Comment:
I might recommend using `black` or `#000000` here instead of `var(--black)` since the `--black` CSS variable may not be a valid CSS variable once the design tokens implementation ships; it'd be replaced by something like `--pgn-color-black` instead.